### PR TITLE
Record Degreed API calls

### PR DESCRIPTION
**Pull request by MueezKhan246**
Description:
Changes to degreed client to record degreed api calls in DB.

JIRA:
https://2u-internal.atlassian.net/jira/software/c/projects/ENT/issues/ENT-8371

### CODE REVIEW

File: integrated_channels/degreed/client.py [hunk 1]
```diff
@@ -42,6 +42,9 @@ def __init__(self, enterprise_configuration):
         self.global_degreed_config = apps.get_model('degreed', 'DegreedGlobalConfiguration').current()
         self.session = None
         self.expires_at = None
+        self.IntegratedChannelAPIRequestLogs = apps.get_model(
+            "integrated_channel", "IntegratedChannelAPIRequestLogs"
+        )
```
Comment (hamzawaleed01) [inline integrated_channels/degreed/client.py:47]: Are we intentionally setting this here? 🤔

Comment (MueezKhan246) [reply integrated_channels/degreed/client.py]: @hamzawaleed01 it was redundant, removed it.

**Review by justEhmadSaeed — APPROVED** [review]
lgtm

**Review by zamanafzal — APPROVED** [review]

**Review by hamzawaleed01 — APPROVED** [review]

File: integrated_channels/degreed/client.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
```
Comment (sameenfatima78) [inline integrated_channels/degreed/client.py:13]: I think the changes in Degreed channel can be discarded as we no longer use this code.

**Review by sameenfatima78 — APPROVED** [review]

File: integrated_channels/degreed/client.py (hunk 2 above)
Comment (MueezKhan246) [reply integrated_channels/degreed/client.py]: @sameenfatima78 reverted those changes

**Pull request merged**